core/src/main/java/org/elasticsearch/rest/action/cat/RestTemplatesAction.java

```diff
@@ -85,7 +87,7 @@ private Table buildTable(RestRequest request, ClusterStateResponse clusterStateR
             if (patternString == null || Regex.simpleMatch(patternString, indexData.name())) {
                 table.startRow();
                 table.addCell(indexData.name());
-                table.addCell(indexData.getTemplate());
+                table.addCell(String.format(Locale.ROOT, "[%s]", String.join(", ", indexData.getTemplate())));
```

Comment:
I'd just do `"[" + String.join(", ", indexData.getTemplate()) + "]"`. I was a big fan of `String.format` for a while but I've fallen out of love. And the compiler is quite good at optimizing `+` for Strings.